website/docs/software_partners/how_to_guides/work_with_onboarding_sessions/index.md

```diff
@@ -0,0 +1,98 @@
+
+```bash
+curl -X POST https://api.superapi.com.au/api/v1/onboarding-session \
+  -H "Content-Type: application/json" \
+  -H "x-api-key: superapi_yourapikeysDZFUnrDIyNp7YTAPDcJXge" \
+  -d '{
+    "employer": {
+      "id": "a46556e4-c7a6-40ec-be37-642c447df460"
+    },
+    "employee": {
+      "remote_id": "1234"
+    },
+    "email": "employee@example.com",
+    "email_validated": true,
+    "phone": "0405 123 456",
+    "phone_number_validated": false,
+    "workflow_slug": "your_provided_slug"
+  }'
+```
+
+This is the minimal payload that can be passed for an onboarding session. That is, the employer that is creating the onboarding session and some basic contact details for the user.
+
+::: tip
+If you have validated the users email address or phone number already, please tell us in the payload.
+:::
+
+## Passing additional details
+
+It is possible to pass additional details when creating an onboarding session. For example, passing an `employee_detail` payload will prefill many of the fields in our system and ensure that the user does not have to double enter details about themselves. It is perfectly safe to pass only as much detail as you have, we will fill in the rest.
+
+```bash
+curl -X POST https://api.superapi.com.au/api/v1/onboarding-session \
+  -H "Content-Type: application/json" \
+  -H "x-api-key: superapi_yourapikeysDZFUnrDIyNp7YTAPDcJXge" \
+  -d '{
+    "employer": {
+      "id": "a46556e4-c7a6-40ec-be37-642c447df460",
+      "name": "The new company name"
+    },
+    "employee": {
+      "remote_id": "1234"
+    },
+    "employee_detail": {
+      "data": %{
+        "address": {
+          "address_line_1": "10 The Street",
+          "locality": "Brunswick",
+          "postcode": "3041",
+          "state": "nsw"
+        },
+        "identity": %{
+          "date_of_birth": "2003/11/09",
+          "family_name": "Smith",
+          "gender": "male"
+        },
+        tax_detail: %{
+          "tfn": "123456789",
+        }
+      }
+    },
+    "email": "employee@example.com",
+    "email_validated": true,
+    "phone": "0405 123 456",
+    "phone_number_validated": false,
+    "workflow_slug": "your_provided_slug"
+  }'
+```
+
+Notice that you can also pass additional details about the employer here. If passed, we will automatically update the employer with those details. This can provide an easy hook to ensure that the employer is always up to date in our system before a user is onboarded to them. After the onboarding session has been created, it will then return a payload of data back to you. The important thing in this payload will be the id, as this is then used to construct the embed which can be given to the user to complete the onboarding.
```

Comment:
```suggestion
Notice that you can also pass additional details about the *employer* here. If passed, we will automatically update the employer with those details. This can provide an easy hook to ensure that the employer is always up to date in our system before a user is onboarded to them. After the onboarding session has been created, it will then return a payload of data back to you. The important thing in this payload will be the id, as this is then used to construct the embed which can be given to the user to complete the onboarding.
```

Was confused for a sec about Employee vs. Employer